.goreleaser-linux-amd64.yml

```diff
@@ -34,6 +34,25 @@ builds:
     goarch:
       - amd64
 
+nfpms:
+  - package_name: confluent
+    builds:
+      - confluent-linux-amd64-disableupdates
+    vendor: Confluent Inc.
+    homepage: https://docs.confluent.io/confluent-cli/current/overview.html
+    maintainer: Confluent Inc.
+    description: CLI for Confluent Cloud and Confluent Platform
+    license: Confluent Community License Agreement
```

Comment:
Is it common to put a link here? Maybe we can link to the actual license in our repo?

Reply:
I looked at a number of repos and nobody is doing it.

But it's just a string, so I don't see why not.

Reply:
> I looked at a number of repos and nobody is doing it.

Let's keep it as is then

Reply:
I changed it to an url and them reverted it when I saw your latest comment, lol.